Makefile

```diff
@@ -105,18 +105,21 @@ build: ## By default we should build in release mode
 
 .PHONY: build-no-std
 build-no-std: ## Build without the standard library
-	${BUILD_KERNEL_ERRORS} cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib
+	${BUILD_KERNEL_ERRORS} cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib --exclude miden-proving-service --features tx-prover
 
 
 .PHONY: build-no-std-testing
 build-no-std-testing: ## Build without the standard library. Includes the `testing` feature
-	cargo build --no-default-features --target wasm32-unknown-unknown --workspace --exclude miden-bench-tx --exclude miden-tx-prover --features testing
+	cargo build --no-default-features --target wasm32-unknown-unknown --workspace --exclude miden-bench-tx --exclude miden-proving-service --features testing,tx-prover
```

Comment:
Probably not related to this PR, but it seems that this should be the same as `build-no-std` except for the `testing` feature, right? But this target seems to build executables as well, which the other one does not (ie, it includes `--lib`).

Reply:
Yes, it is quite odd because it manually excludes both binaries. For the reason mentioned above I don't think that we can remote the `miden-proving-service` exclusion but we can replace the `--exclude miden-bench-tx` with `--lib`.

Reply:
It is quite a simple change, and i do think that it is more _correct_ that way. I can push it if we want to include it.